### ISSUE TITLE
PDFs im beA

### ISSUE THREAD

**Issue by Dickmaulruessler** [jlawyerorg/j-lawyer-org#713]
Hallo, 

ich finde die aktuelle Ansicht der Anhänge nicht übersichtlich genug. Ich würde es besser finden, wenn jeder Anhang in eine **eigene Zeile** käme, also alle untereinander, nicht nebeneinander. 

Ist auch eine **PDF-Vorschau** innerhalb von jlawyer möglich? Also, dass man mit einem einzelnen Klick die PDF (verkleinert) direkt in einem bestehenden jlawyer-Fenster sehen kann und sie sich erst nach einem Doppelklick in einem gesonderten (großen) PDF-Fenster öffnet...?

Dank an den Chef!

**Comment by j-dimension** [comment]
Herzlich willkommen und Danke für Dein erstes Ticket! :-)

> ich finde die aktuelle Ansicht der Anhänge nicht übersichtlich genug. Ich würde es besser finden, wenn jeder Anhang in eine **eigene Zeile** käme, also alle untereinander, nicht nebeneinander.

Wenn die Gerichte anfangen, Akten zur Einsicht per beA zur Verfügung zu stellen, dann werden die Anlagenanzahlen sehr hoch sein. Die "Testakten" die ich diesbezüglich erhalten habe, waren jenseits der 40 PDFs. Da ist man dann ganz schön am Scrollen. Auch wäre dann eine Menge ungenutzter Bildschirmplatz. Den müsste man - wenn man ihn nicht verschwenden will - sinnvoll nutzen und zu jedem Anhang Zusatzinformationen einblenden.



> Hallo,
> 
> ich finde die aktuelle Ansicht der Anhänge nicht übersichtlich genug. Ich würde es besser finden, wenn jeder Anhang in eine **eigene Zeile** käme, also alle untereinander, nicht nebeneinander.
> 
> Ist auch eine **PDF-Vorschau** innerhalb von jlawyer möglich? Also, dass man mit einem einzelnen Klick die PDF (verkleinert) direkt in einem bestehenden jlawyer-Fenster sehen kann und sie sich erst nach einem Doppelklick in einem gesonderten (großen) PDF-Fenster öffnet...?
> 

Ja, das gibt es zum Beispiel im Scaneingang oder in der Akte. Wenn Du das an zusätzlichen Orten haben willst, gern angeben wo genau.

**Comment by Dickmaulruessler** [comment]
> Wenn Du das an zusätzlichen Orten haben willst, gern angeben wo genau.

unter "beA Anwaltspostfach", dort beim "Posteingang"

**Comment by librelegal** [comment]
Ist dies evtl. ein 
Duplicate of #354 ?

**Comment by j-dimension** [comment]
Ich würde gern #354 #713 #63 getrennt umsetzen - es betrifft unterschiedliche Bereiche des Clients, die alle auch unterschiedliche Nutzungsfrequenzen haben.

**Comment by Dickmaulruessler** [comment]
Ich habe noch etwas anzumerken zum **"wandern" mit den Pfeiltasten durch die Dokumentenliste**, um sich so einen schnellen Überblick über die einzelnen Dokumente zu verschaffen: 

Wenn man sich mit den Pfeiltasten durch die Dokumentenliste klickt, wird dies bei den beA-Nachrichten dadurch gestört, dass man dort dann keine PDF-Vorschau sieht. Aber selbst wenn es diese gäbe, wäre noch unklar, welcher Anhang (bei gesendeten / bei emfpangenen beA-Nachrichten) automatisch als erstes in der PDF-Vorschau angezeigt werden würde. Hättest du dafür schon eine Idee, Jens?

**Comment by Dickmaulruessler** [comment]
Du hast immer wirklich gute Tipps!
Das werde ich so standardmäßig machen. Merci!

**Comment by iradraconis** [comment]
Fast hätte ich ein neues Ticket Duplikat aufgemacht. Ich denke, dass das hier rein passt:

Ich bekomme jeden Tag zahlreiche Emails mit solchen Anhängen. Wenn beim Zuordnen der Mail zu einer Akte und dem Benennen der zu speichernden Datei eine Vorschau der Datei angezeigt würde, wüsste der Nutzer direkt, was da eigentlich abgespeichert wird. So muss der Nutzer nicht noch einmal in die Akte gehen, um dort die Dateien nochmal zu sichten, um sie "sprechend" benennen zu können. Gerade bei Mails mit mehr als 5 Anhängen wird es schnell mühselig. Vielleicht kann die Vorschau da Abhilfe schaffen :) 

![Bildschirmfoto 2022-07-21 um 14 20 01](https://user-images.githubusercontent.com/71747937/180213237-0947d244-910a-4410-b3f0-de8c90e05303.png)

**Comment by j-dimension** [comment]
@iradraconis fände ich vom Workflow her tatsächlich eine gute Lösung!

**Comment by stm9x9** [comment]
ich würde auch nochmal gerne auf den Layout-Vorschlag für die Anhänge hinweisen :-D
https://github.com/jlawyerorg/j-lawyer-org/issues/713#issuecomment-779912936

**Comment by Dickmaulruessler** [comment]
> 
> Ich bekomme jeden Tag zahlreiche Emails mit solchen Anhängen. Wenn beim Zuordnen der Mail zu einer Akte und dem Benennen der zu speichernden Datei eine Vorschau der Datei angezeigt würde, wüsste der Nutzer direkt, was da eigentlich abgespeichert wird. So muss der Nutzer nicht noch einmal in die Akte gehen, um dort die Dateien nochmal zu sichten, um sie "sprechend" benennen zu können. Gerade bei Mails mit mehr als 5 Anhängen wird es schnell mühselig. Vielleicht kann die Vorschau da Abhilfe schaffen :)
> 

Ja, eine PDF-Vorschau ist wirklich - in allen denkbaren Bereichen - unheimlich wertvoll. 
Mit dem beA wird das nun sogar auch ausgehend von den Gerichten immer häufiger, so dass man hier mit dem beA-Posteingang nun wirklich permanent damit zu tun hat.

**Comment by Esko84DO** [comment]
Wie wäre es, wenn man das Vorschaufenster verschiebbar macht (nach Wunsch innerhalb des bisherigen Fensters oder auch daneben).

Damit wären doch viele Probleme lösbar: Anzeige der Anhänge von beA/Mails und ggf. auch bei der Auswahl eines Dokumenten-Etiketts auf dem Desktop. Und sicher noch zahllosen anderen Stellen...
